CalculatorApp/Controller/ViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ViewController.swift
+//  CalculatorApp
+//
+//  Created by Memory Mhou on 04/03/2024.
+//
+
+import UIKit
+
+
+class ViewController: UIViewController {
+    
+ @IBOutlet weak var OutPutLabel: UILabel!
```

Comment:
please read this task on trello regaridng how to name variables and what should have a capital letter and what shouldn't : https://trello.com/c/iaqIpt4P/33-ios-naming-https-developerapplecom-library-archive-documentation-cocoa-conceptual-codingguidelines-articles-namingmethodshtml

Reply:
i have updated the variables names.